### PR TITLE
Enable tests

### PR DESCRIPTION
**Pull request by ngaruko**
- Enable most of tests
-added helper function to alter timeouts and handle exceptions on click() and getText()
-used url for some page navigations instead of click()

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "simple-password-tester": "^1.0.0",
     "underscore": "^1.8.3",
     "uuid": "^3.1.0",
+    "webdriverjs-retry": "^0.2.0",
```
Comment (garethbowen) [inline package.json]: Firstly this should be in "devDependencies" not "dependencies". Secondly, I'm not sure it's used at all?

File: tests/protractor/conf.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ const startApi = () => startNodeModule('api', 'Medic API listening on port');
 
 const startSentinel = () => startNodeModule('sentinel', 'startup complete.');
 
-// start sentinel serially because it relies on api.
+// start sentinel serially because it relies on api..
```
Comment (garethbowen) [inline tests/protractor/conf.js]: Can you revert this comment change and all the others below that are similar? It makes viewing historical changes later more difficult.

File: package.json (hunk 1 above)
Comment (ngaruko) [reply package.json]: will remove this

Comment (garethbowen) [reply package.json]: Cool. Be sure to revert the package-lock.json file too...

**Pull request merged**